tasks/main.yml

```diff
@@ -10,12 +10,18 @@
     name: nginx-full
   when: ansible_pkg_mgr == "apt"
   
-- name: set nginx config
+- name: Set nginx config
   template:
     src: templates/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
   register: nginx_reload
 
+- name: Place template pam nginx file
+  template:
+    src: "templates/pam_service_file.j2"
+    dest: "/etc/pam.d/{{ nginx_pam_service_name }}"
```

Comment:
documentation? Is it mandatory? It should be, or its definition should be checked.

Reply:
see the when statment below